### DISCUSSION TITLE
ChromiumWebBrowser.IsBrowserInitialized  一直处于False

### DISCUSSION THREAD

**Discussion by YuNingOvo** [cefsharp/CefSharp#4211] [General]
![image](https://user-images.githubusercontent.com/31461209/184864553-67030e64-d1c3-40c9-a5f2-55e80cc90ad5.png)

**Comment by YuNingOvo** [comment]
控制台中调用。

**Reply by amaitland** [reply]
If you still require assistance then please ask your question in English, also you'll need to provide a mode detailed code example, [no images please](https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#no-images-of-code).

Please answer as many questions from https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following as you can
